src/components/API/MenuResults.jsx

```diff
@@ -27,59 +24,85 @@ export function MenuResults() {
       getOrder();
     }
 
+    const [data, setData] = useState([]);
```

Comment:
data and filteredData are declared twice, which will result in an error.

Reply:
Thanks for pointing that out! Fixed and verified merged branch is running without errors!